src/App.tsx

```diff
@@ -15,13 +14,18 @@ import AppNavigator from 'src/navigation/AppNavigator';
 import ThemeProvider from './context/ThemeContext';
 import {IonicIconsPack} from './Ionic-icons';
 import {NavigationContainer} from 'context/NavigationContainer';
+import SplashScreen from 'react-native-splash-screen';
 
 // init type registry
 import 'src/typeRegistry';
 
 const queryClient = new QueryClient();
 
 export default function App() {
+  React.useEffect(() => {
+    SplashScreen.hide();
```

Comment:
It would be better to hide it after the navigation is ready in our NavigationContainer:

```
<RNnavigationContainer onReady={() => SplashScreen.hide()} linking={linking} theme={theme === 'dark' ? darkTheme : lightTheme}>
   {children}
</RNnavigationContainer>
```

Reply:
done ✅